contrib/example-extension-package/mybots/bots/collectors/custom/collector.py

```diff
@@ -0,0 +1,21 @@
+"""
+SPDX-FileCopyrightText: 2023 CERT.at GmbH <https://cert.at/>
+SPDX-License-Identifier: AGPL-3.0-or-later
+"""
+
+from intelmq.lib.bot import CollectorBot
```

Comment:
The example could additionally show, that import from a own library can be done like this:
```suggestion
from intelmq.lib.bot import CollectorBot
from mybots.lib import common_code
```
or alike